python/hail/dataset.py

```diff
@@ -14,6 +14,15 @@
 class VariantDataset(object):
     """Hail's primary representation of genomic data, a matrix keyed by sample and variant.
 
+    Variant Datasets can be generated from a VCF file with :py:meth:`~hail.HailContext.import_vcf`,
```

Comment:
I think this capitalization looks bad.  Either use code:

A `VariantDataset` can be generated from...

However, I think we can tighten all this up.  Basically everything on HailContext creates a VDS, so we should just link there.